src/main/java/Wolox/training/controllers/BookController.java

```diff
@@ -1,16 +1,55 @@
 package Wolox.training.controllers;
 
+import Wolox.training.exceptions.BookNotFoundException;
+import Wolox.training.models.Book;
+import Wolox.training.repositories.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
 
 @Controller
+@RequestMapping("/api/books")
```

Comment:
You can declare through a property the context path (`server.servlet.context-path=/api`), so you avoid repeating the word in each controller.
```suggestion
@RequestMapping("/books")
```